joomla/joomla-cms
# Add tooltip of modified by

#### Summary of Changes

Added tooltip for "Modified by" field in Banner, Contact and Newsfeed page.
#### Testing Instructions
- Login at administrator side.
- Check banner, contact and newsfeed new/edit page->Publishing tag ->Modified by field.
  There is no tooltip for this field.
  Apply this PR and follow above steps, you can see tooltip for this field.
